table/shell/src/main/java/alluxio/cli/table/command/AbstractTableCommand.java

```diff
@@ -36,8 +39,11 @@
    *
    * @param conf the alluxio configuration
    * @param client the client interface which can be used to make RPCs against the table master
+   * @param fsContext the filesystem of Alluxio
    */
-  public AbstractTableCommand(AlluxioConfiguration conf, TableMasterClient client) {
+  public AbstractTableCommand(AlluxioConfiguration conf, TableMasterClient client,
+      @Nullable FileSystemContext fsContext) {
```

Comment:
Why is nullable supported here? Shouldn't all commands be able to take a non-null context?

Reply:
I agree